src/test/cypress/e2e/exam/test-exam/TestExamTestRun.cy.ts

```diff
@@ -0,0 +1,154 @@
+import { admin, instructor } from '../../../support/users';
+
+// Common primitives
+const textFixture = 'loremIpsum.txt';
+const examTitle = 'exam' + generateUUID();
+
+const exerciseArray: Array<Exercise> = [];
+
+describe('Test exam test run', () => {
+    let course: Course;
+    let exam: Exam;
+    let testRun: any;
+
+    before(() => {
+        cy.login(admin);
+        courseManagementRequest.createCourse(true).then((response) => {
+            course = convertCourseAfterMultiPart(response);
+            const examContent = new CypressExamBuilder(course)
+                .title(examTitle)
+                .testExam()
+                .visibleDate(dayjs().subtract(3, 'days'))
+                .startDate(dayjs().add(1, 'days'))
+                .endDate(dayjs().add(3, 'days'))
+                .examMaxPoints(40)
+                .numberOfExercises(4)
+                .build();
+            courseManagementRequest.createExam(examContent).then((examResponse) => {
+                exam = examResponse.body;
+                examExerciseGroupCreation.addGroupWithExercise(exerciseArray, exam, EXERCISE_TYPE.Text, { textFixture });
+
+                examExerciseGroupCreation.addGroupWithExercise(exerciseArray, exam, EXERCISE_TYPE.Programming, { submission, practiceMode: true });
+
+                examExerciseGroupCreation.addGroupWithExercise(exerciseArray, exam, EXERCISE_TYPE.Quiz, { quizExerciseID: 0 });
+
+                examExerciseGroupCreation.addGroupWithExercise(exerciseArray, exam, EXERCISE_TYPE.Modeling);
+            });
+        });
+    });
+
+    beforeEach('Create test run instance', () => {
+        cy.login(instructor);
+        // API call does not work yet, for now we use the UI to create the test run
+        // courseManagementRequest.createExamTestRun(exam, exerciseArray).then((response: any) => {
+        //     testRun = response.body;
+        // });
+        cy.visit(`/course-management/${course.id}/exams/${exam.id}`);
+        examManagement.openTestRun();
+        examTestRun.createTestRun();
+        examTestRun.confirmTestRun().then((testRunResponse: Interception) => {
+            testRun = testRunResponse.response!.body;
+        });
+    });
+
+    it('Create a test run', () => {
+        cy.login(instructor);
+
+        let testRunID: number;
+        const minutes = 40;
+        const seconds = 30;
+
+        cy.visit(`/course-management/${course.id}/exams/${exam.id}`);
+        examManagement.openTestRun();
+        examTestRun.createTestRun();
+        examTestRun.setWorkingTimeMinutes(minutes);
+        examTestRun.setWorkingTimeSeconds(seconds);
+        examTestRun.confirmTestRun().then((testRunResponse: Interception) => {
+            const testRun = testRunResponse.response!.body;
+            testRunID = testRun.id;
+
+            expect(testRunResponse.response!.statusCode).to.eq(200);
+            expect(testRun.testRun).to.eq(true);
+            expect(testRun.submitted).to.eq(false);
```

Comment:
(optional) Is this maybe better?

```suggestion
            expect(testRun.testRun).to.be.true;
            expect(testRun.submitted).to.be.false;
```

Reply:
Good idea!